### PR TITLE
fix manifest to include new webpack-compiled output files

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** we were deploying to pypi in a way that didn't include the new webpack output and it broke everything

**What changed?**

- we used to compile js to `openassessment-lms.min.js` and `openassessment-studio.min.js`
- as of 2.11.0 we compile them to `openassessment-lms.js` and `openassessment-studio.js`
- the manifest wasn't updated so we weren't picking up our comipled files anymore when we deployed to pypi

**Developer Checklist**
- [x] Reviewed the [release process](./release_process.md)
- [x] Translations up to date
- [x] JS minified, SASS compiled
- [x] Test suites passing on Jenkins
- [x] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)

JIRA: [EDUCATOR-5399](https://openedx.atlassian.net/browse/EDUCATOR-5399)

FIY: @edx/masters-devs-gta

### CODE REVIEW

**Review by muselesscreator — APPROVED** [review]
...wow... that was it?!?!
Thanks for finding this!

**Comment by jansenk** [comment]
> ...wow... that was it?!?!
> Thanks for finding this!

I _THINK_ that's it
